Sources/RSDKUtils/Networking/Reachability.swift

```diff
@@ -0,0 +1,275 @@
+import SystemConfiguration
+import Foundation
+#if canImport(Logger)
```

Comment:
We should open source Logger.
What do you think @malayli ?

Reply:
Maybe Logger can be brought into this repo actually

Reply:
Good idea. Especially when canImport() won't work

Reply:
We can open source RLogger or move it and its tests to SDKUtils @donnie-jp @imaginaris

Reply:
But RLogger contains Objective-C code for now. There is a ticket to remove the Objective-C support in RLogger.
Can SDKUtils contain Objective-C code?

Reply:
No lets remove the Obj-C before importing here